sys/include/ztimer.h

```diff
@@ -305,7 +305,7 @@ typedef struct {
      * @param   clock       ztimer clock to set the new target
      * @param   val         Relative target (e.g. fire at value `now() + val`)
      */
-    void (*set)(ztimer_clock_t *clock, uint32_t val);
+    void (*set)(ztimer_clock_t * clock, uint32_t val);
```

Comment:
shouldn't the proper style be
```suggestion
    void (*set)(ztimer_clock_t* clock, uint32_t val);
```

Reply:
Hmm, I am not sure, everywhere else the star is put before the parameter name (even in the case of the third function in this struct, see line 314). What is very odd is the fact that the star placement seems to only fail in the case of functions with return type `void`. If you change the return type from `void` to `uint32_t`, for example, the uncrustification works as expected 🤔

Reply:
Uh yea of course - `ztimer_clock_t *clock` is correct, not `ztimer_clock_t* clock`.
Not sure why uncrustify insists on putting a space here.

Reply:
I just did some investigation and noticed that the uncrustify package [used in the riotdocker image](https://github.com/RIOT-OS/riotdocker/runs/2773139817?check_suite_focus=true#step:5:803) (and installed under ubuntu using the default package sources) is version 0.66.0 which dates back to 2017 (current version is 0.73.0. Updating uncrustify to the latest version should probably fix this issue. I guess we can leave this as is for now as a reminder?

Reply:
https://github.com/RIOT-OS/riotdocker/pull/139 should help

Reply:
Oh, very nice! :) @kaspar030: Can you estimate when that PR will be ready?

Reply:
I guess someone needs to review it ;)

Reply:
With the new uncrustify version the original version of this line does not get flagged anymore :) I therefore reverted the change.